cluster-dns-connector/src/main/java/com/sequenceiq/cloudbreak/client/GrpcClusterDnsClient.java

```diff
@@ -102,6 +103,19 @@ public DeleteDnsEntryResponse deleteDnsEntryWithCloudDns(String accountId, Strin
         }
     }
 
+    public GenerateManagedDomainNamesResponse generateManagedDomain(String environmentName, List<String> subDomains, String accountId,
+            Optional<String> requestId) {
+            String requestIdValue = requestId.orElse(UUID.randomUUID().toString());
```

Comment:
bad indentation

Reply:
@sodre90 I am resolving it as already fixed and GH marked your comment as `Outdated`.